api_specs/v2/component_analyses.yaml

```diff
@@ -106,7 +143,122 @@ paths:
         404:
           description: Data not found
           content: {}
+
 components:
+  examples:
+    free-user-batch:
+      value:
+        - recommendation:
+            component_analyses:
+              vulnerability:
+                - vendor_cve_ids: "CVE-2010-3082"
```

Comment:
Vendor cve id is not actual cve id. It starts with SNYK-..

Reply:
Ack!